docs/reference/dedicated-server.md

```diff
@@ -0,0 +1,89 @@
+---
+id: dedicated-server
+title: Dedicated Game Server (DGS)
+sidebar_label: Dedicated Game Server
+description: NEW DESCRIPTION
+---
+
+## What is a dedicated server?
+
+A dedicated server is a standalone, headless platform build optimized for running in the cloud as a game server application. They simulate game worlds without supporting direct input or output, except that required for their administration. Players must connect to the server with separate client programs to see and interact with the game.
+
+![Dedicated Server](/img/ded_server.png)
+
+Many multiplayer games rely on a client-host (a listen server setup) as both the client and server. Because it relies on the host machine's internet connection and hardware capabilities, its simplistic setup can cause performance issues such as lag, slow loading times, crashes, loss of save files, and dropped games. A dedicated server is often a more stable and enjoyable gaming experience.
+
+A dedicated server manages all hosting duties, such as: player locations and activity, games rules, physics, weapons, and equipment. However, the dedicated server does not run the game engine or render anything. These functions (such as graphics, game controls, and sound) is responsible on the client's machine. The dedicated server _instructs_ the client machines on what to render and when.
+
+## Advantages
+
+* Greater processing, memory, and storage capabilities relative to other hosting solutions
+* Facilitates a reliable, stable, and faster connection
+* Often eliminates lag, crashes, or glitches
+* Can handle high traffic
+* Less prone to corrupt save files
+* Higher levels of security based on admin login definitions and the server's sole purpose to run the game
+
+## Disadvantages
+
+* Setup is more complex than other network topologies
```

Comment:
The complexity is not that much related to the network topology. I don't think that is that much different from a host in this regard. 
Complexity is mostly about the difficulty to manage two distinct projects : 1 for the player, 1 for the server.